### PR TITLE
Obfuscating Loop1

### PR DESCRIPTION
**Pull request by jcbeyler**
The loop1 method is too simple for LLVM, which then makes it return just 5.
Though this proves that the code is correct, we probably want to have the
  code generation all the same.

This patch makes the code a bit different but at least the compiler can no
  longer optimize it all away.

### CODE REVIEW

File: ml-proto/test/labels.wast [hunk 1]
```diff
@@ -6,12 +6,12 @@
...
 
-  (func $loop1 (result i32)
+  (func $loop1 (param $max i32) (result i32)
     (local $i i32)
-    (set_local $i (i32.const 0))
+    (set_local $i (i32.const 1))
     (loop $exit $cont
-      (set_local $i (i32.add (get_local $i) (i32.const 1)))
-      (if (i32.eq (get_local $i) (i32.const 5))
+      (set_local $i (i32.add (get_local $i) (get_local $i)))
```
Comment (rossberg-chromium) [inline ml-proto/test/labels.wast]: This is doubling $i on each iteration, so the expected result would be something like 2^8, no?

Comment (rossberg-chromium) [inline ml-proto/test/labels.wast]: Ah, sorry, misread break condition.

**Comment by rossberg-chromium** [comment]
LGTM

**Comment by sunfishcode** [comment]
What would you think about just adding the new more complex loop as a new test, rather than replacing the existing test? I think there's value in both.

**Comment by jcbeyler** [comment]
Done, I added a loop3 then

**Comment by jcbeyler** [comment]
let me do this again...

**Pull request closed without merging**